Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "mitmguard"
+name = "mitmproxy_wireguard"
```

Comment:
I did like the "mitmguard" combination a lot, but this boring one has the big benefit that it's self-explanatory.